utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/GeneratePlatformLibraries.kt

```diff
@@ -83,6 +83,7 @@ private fun generatePlatformLibraries(target: String, inputDirectory: File, outp
         if (!saveTemps)  {
             File("$outputDirectory/build-${def.name}").deleteRecursively()
         }
+        File("$outputDirectory/clangModulesCache").deleteRecursively()
```

Comment:
The idea behind module cache here was to reuse it for all platform libraries.

Reply:
Yeah, moved removal below, PTAL.